src/common/dlrgint.c

```diff
@@ -25,10 +25,9 @@ dptr da;
 /*
  *  bignum -> real
  */
-double bigtoreal(da)
-dptr da;
+int bigtoreal(dptr da, double *result)
```

Comment:
It seems slightly odd to have the modern syntax for the parameter types, when the rest of the file has the "old school" way of specifying them. Especially as its counterpart (in src/runtime/rlrgint.r line 277) has also been modified, but retains the old school syntax.

Reply:
I usually migrate to the new style when possible, i.e, if I  touch something old I update to the new style. I will fix the other one.